src/PlaylistContainer/PlaylistContainer.tsx

```diff
@@ -17,7 +17,7 @@ enum PlaylistTypes {
 	// CustomPlaylist = 'custom-playlist',
 }
 
-const displayPlaylist = (playlistType: string, playlists: [CleanedAlbumTrack[]] | [], selectedGenre: string) => {
+const displayPlaylist = (playlistType: string, playlists: any, selectedGenre: string) => {
```

Comment:
The bug with Typescript having issues iteration over array in specific instances when a union is used made it quite difficult to figure this out. We have both checks before and after to ensure that we are using our CleanedAlbumTrack interface. While this may not necessarily be ideal, having any in specific situations may be the right call. We can do more research on it but it being a known bug with typescript and having checking before and after that are validating the data that we want feels okay.